pkg/sfu/streamallocator.go

```diff
@@ -20,7 +20,19 @@ import (
 const (
 	ChannelCapacityInfinity = 100 * 1000 * 1000 // 100 Mbps
 
-	EstimateEpsilon = 2000 // 2 kbps
+	NumRequiredEstimatesNonProbe = 8
```

Comment:
Lots of change in this file, but the most important bits are
1. Probing with padding (interesting functions to look there are `maybeProbeWithPadding`, `handleNewEstimateInProbe`, `onProbeClusterDone` and `finalizeProbe`)
2. The new `Estimator` towards the end of this file's diff. Using a trend indicator (simple one using Kendall's Tau) to figure which way things are going and how to react. I think that is an elegant way to handle this (although I need to keep hammering on this to ensure that there are no gotchas, so far has been working very well)

Reply:
yes yes! using a trend indicator makes a ton of sense here. excited to have this change!